pkg/trace/api/otlp.go

```diff
@@ -239,7 +240,7 @@ func (o *OTLPReceiver) ReceiveResourceSpans(rspans ptrace.ResourceSpans, header
 		containerID = rattr[string(semconv.AttributeK8SPodUID)]
 	}
 	if containerID == "" {
-		containerID = fastHeaderGet(header, headerContainerID)
```

Comment:
@gbbr Was there an important performance gain with `fastHeaderGet` that we want to retain?